audio-worklet/src/bitcrusher-audio-worklet.js

```diff
@@ -40,15 +40,16 @@ registerProcessor('bitcrusher-audio-worklet',
   constructor(options) {
     super(options);
 
-    // Index_ and previousSample_ are member variables to handle block
-    // transitions.
+    // index_ and previousSample_ are member variables to handle block
+    // transitions. index_ refers to the current position in the input
+    // AudioBuffer, and previousSample_ refers to the corresponding sample.
```

Comment:
I'm going to leave it as it is so that it has a clear correspondance to scriptprocessor implementation.